langchain4j-qianfan/pom.xml

```diff
@@ -66,6 +74,31 @@
         </dependency>
     </dependencies>
 
+    <build>
+        <pluginManagement>
+            <plugins>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-enforcer-plugin</artifactId>
+                    <version>3.5.0</version>
+                    <executions>
+                        <execution>
+                            <id>enforce</id>
+                            <configuration>
+                                <rules>
+                                    <dependencyConvergence/>
+                                </rules>
+                            </configuration>
+                            <goals>
+                                <goal>enforce</goal>
+                            </goals>
+                        </execution>
+                    </executions>
+                </plugin>
```

Comment:
It makes sense to move this to parent pom and use [`enforcer.skipRules`](https://maven.apache.org/enforcer/maven-enforcer-plugin/enforce-mojo.html#rulesToSkip) property to skip rules for some modules meanwhile. Eventually removing`enforcer.skipRules`.